src/views/jobs/jobs.jsx

```diff
@@ -31,16 +31,16 @@ var Jobs = React.createClass({
                         <h3><FormattedMessage id='jobs.openings' /></h3>
                         <ul>
                             <li>
-                                <a href="http://bit.ly/2f5hABE">
-                                    Trust &amp; Safety Manager
+                                <a href="/jobs/moderator">
+                                    Community Moderator
                                 </a>
                                 <span>
-                                    (MIT Media Lab, Cambridge, MA)
+                                    MIT Media Lab, Cambridge, MA (or Remote)
```

Comment:
I think it'd be good to make this be `(MIT Media Lab, Cambridge, MA or Remote)` to match with the parentheses of the Trust & Safety Manager